### ISSUE TITLE
Exception occurred when attempting to send a Twilio Phone Call alert

### ISSUE THREAD

**Issue by doboy** [arachnys/cabot#471]
On a fresh installation of master (at the time) I get the following when a service has failed and cabot attempts to send a Twilio Phone Call alert. If anyone has any insight on why this is occurring for me that would be helpful! Let me know if there is any more information I should provide.

```
$ tail -f celery-1.log
Traceback (most recent call last):
  File "/home/ubuntu/2017-03-22-9f8291b/cabot/cabotapp/alert.py", line 42, in send_alert
    alert.send_alert(service, users, duty_officers)
  File "/home/ubuntu/venv/local/lib/python2.7/site-packages/cabot_alert_twilio/models.py", line 31, in send_alert
    reverse('twiml-callback', kwargs={'service_id': service.id}))
  File "/home/ubuntu/venv/local/lib/python2.7/site-packages/django/urls/base.py", line 91, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/home/ubuntu/venv/local/lib/python2.7/site-packages/django/urls/resolvers.py", line 392, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns)
NoReverseMatch: Reverse for 'twiml-callback' with arguments '()' and keyword arguments '{'service_id': 5}' not found. 0 pattern(s) tried: []
```

**Comment by frankh** [comment]
Hi @doboy 
It looks like maybe the twilio plugin didn't install properly...
If you run pip freeze does it come up?

**Comment by doboy** [comment]
Thank you for your response, here is the output

```
(venv)ubuntu@cabot-0:~/venv/bin$ pip freeze
Django==1.10.6
PySocks==1.6.6
amqp==2.1.4
anyjson==0.3.3
appdirs==1.4.3
argparse==1.2.1
billiard==3.5.0.2
-e git+git@github.com:arachnys/cabot.git@9f8291b54eba2c062ba331639092773f3dffb767#egg=cabot-master
cabot-alert-email==1.4.3
cabot-alert-hipchat==1.8.3
cabot-alert-slack==0.7.0
cabot-alert-twilio==1.3.0
celery==4.0.2
dj-database-url==0.4.2
django-appconf==1.0.2
django-auth-ldap==1.2.10
django-compressor==2.1.1
django-filter==1.0.1
django-jsonify==0.3.0
django-polymorphic==1.1
djangorestframework==3.5.4
freezegun==0.3.8
gevent==1.2.1
greenlet==0.4.12
gunicorn==19.7.0
httplib2==0.10.3
icalendar==3.11.3
kombu==4.0.2
packaging==16.8
psycopg2==2.7.1
pyldap==2.4.28
pyparsing==2.2.0
python-dateutil==2.6.0
python-ldap==2.4.32
pytz==2016.10
rcssmin==1.0.6
redis==2.10.5
requests==2.13.0
rjsmin==1.0.12
six==1.10.0
twilio==5.7.0
vine==1.1.3
whitenoise==3.3.0
wsgiref==0.1.2
```

**Comment by frankh** [comment]
Can you check if the `CABOT_PLUGINS_ENABLED` environment variable is being set properly?

it should be set to something like `cabot_alert_hipchat,cabot_alert_twilio,cabot_alert_email,cabot_alert_slack`, make sure there are no extra spaces or anything

**Comment by doboy** [comment]
It appears that I do not have it within my env

```
(venv)ubuntu@cabot-0:~$ echo $CABOT_PLUGINS_ENABLED

(venv)ubuntu@cabot-0:~$
```

However, my production config has that information

```
(venv)ubuntu@cabot-0:~/cabot/conf$ grep CABOT_PLUGINS_ENABLED production.env
CABOT_PLUGINS_ENABLED=cabot_alert_hipchat==1.6.1,cabot_alert_twilio==1.1.4,cabot_alert_email==1.3.1
```

**Comment by frankh** [comment]
@doboy i've managed to reproduce and confirmed it's a bug on master, working on a fix now

**Comment by frankh** [comment]
In the meantime, it is working on 0.9.2 so you can try installing that tag instead

**Comment by doboy** [comment]
Thank you for your time, do you know which commit caused the regression? I maybe able to help fix it

**Comment by frankh** [comment]
@doboy this is now fixed in 0.10.1

Thanks for reporting this
